aea/protocols/base.py

```diff
@@ -266,90 +290,6 @@ class Serializer(Encoder, Decoder, ABC):
     """The implementations of this class defines a serialization layer for a protocol."""
 
 
-class ProtobufSerializer(Serializer):
```

Comment:
I assume this is not used anywhere now and therefore removed?

Reply:
@marcofavorito  safe to remove?

Reply:
well since we deleted the possibility to serialized `Message` object directly then I'd say yes.